lib/src/source/git.dart

```diff
@@ -411,7 +413,12 @@ class BoundGitSource extends CachedSource {
     var path = _repoCachePath(ref);
     assert(!_updatedRepos.contains(path));
 
-    await _clone(ref.description['url'], path, mirror: true);
+    try {
+      await _clone(ref.description['url'], path, mirror: true);
+    } catch (error, _) {
```

Comment:
`} catch (_) {` should do

Reply:
Maybe give this a one-line comment along the lines of: `// clean-up invalid git repositories after failed git clone`